### ISSUE TITLE
Error initializing railtie

### ISSUE THREAD

**Issue by betelgeuse** [Compass/compass#446]
```
$ bundle exec rails s -b localhost
=> Booting WEBrick
=> Rails 3.0.9 application starting in development on http://localhost:3000
=> Call with -d to detach
=> Ctrl-C to shutdown server
Exiting
/home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/actions.rb:83:in `relativize': undefined method `length' for #<Pathname:/> (NoMethodError)
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/actions.rb:78:in `basename'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/actions.rb:73:in `remove'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/compiler.rb:80:in `clean!'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/compiler.rb:79:in `each'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/compiler.rb:79:in `clean!'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/configuration/helpers.rb:141:in `handle_configuration_change!'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/app_integration/rails.rb:61:in `initialize!'
        from /home/betelgeuse/.gem/ruby/1.8/gems/compass-0.11.3/lib/compass/app_integration/rails/actionpack3/railtie.rb:43
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/initializable.rb:25:in `instance_exec'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/initializable.rb:25:in `run'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/initializable.rb:50:in `run_initializers'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/initializable.rb:49:in `each'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/initializable.rb:49:in `run_initializers'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/application.rb:134:in `initialize!'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/application.rb:77:in `send'
        from /home/betelgeuse/.gem/ruby/1.8/gems/railties-3.0.9/lib/rails/application.rb:77:in `method_missing'
        from /home/betelgeuse/hobo/application/config/environment.rb:5

```

**Comment by gavinlaking** [comment]
Also happening for me and only since I upgraded to Rails 3.0.9.

**Comment by travisp** [comment]
Happening for me on Rails 2.3.8

**Comment by chriseppstein** [comment]
fixed by 852f03316008afe1ccb2106d602a4288c27cc738
